pkg/k8s/api.go

```diff
@@ -166,6 +166,16 @@ func (kubeAPI *KubernetesAPI) GetReplicaSets(ctx context.Context, namespace stri
 	return replicaSetList.Items, nil
 }
 
+// GetAllNamespacesWithExtensionLabel gets all namespaces with the linkerd.io/extension label key
+func (kubeAPI *KubernetesAPI) GetAllNamespacesWithExtensionLabel(ctx context.Context) ([]corev1.Namespace, error) {
+	namespaces, err := kubeAPI.CoreV1().Namespaces().List(ctx, metav1.ListOptions{LabelSelector: LinkerdExtensionLabel})
+	if err != nil {
+		return nil, err
+	}
+
+	return namespaces.Items, nil
+}
+
```

Comment:
`GetAllNamespacesWithExtensionLabel` already exists on `main`. You may want to make sure you're based off the most recent commit.

Reply:
Done